core/array.cpp

```diff
@@ -34,10 +34,11 @@
 #include "core/object.h"
 #include "core/variant.h"
 #include "core/vector.h"
+#include <atomic>
```

Comment:
Style nitpick, but there should be an empty line between our headers `"..."` and system headers `<...>`.
http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
Fixed.